### ISSUE TITLE
archive 18f sites - ads and paid leave proto

### ISSUE THREAD

**Issue by amirbey** [18F/federalist#2509]
#### User Story

<TODO>

#### Background (Optional)

<TODO>

#### Acceptance Criteria

- [ ] <TODO>
- [x] https://paid-leave-prototype.18f.gov redirects to repository
- [x] https://ads.18f.gov redirects to repository

_After evaluating, edit this part:_

#### Implementation outline (if higher than "low" effort):
- [x] redirect https://paid-leave-prototype.18f.gov to repository
- [x] redirect https://ads.18f.gov to repository

**Comment by amirbey** [comment]
@afeld  - instead, this could now be done using 18f/dns redirect_to  functionality?

**Comment by afeld** [comment]
While [S3](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/redirect-website-requests.html) (and thus [Terraform](https://www.terraform.io/docs/providers/aws/r/s3_bucket.html#redirect_all_requests_to) and thus [the module](https://github.com/mediapop/terraform-aws-redirect#inputs)) asks for the "target bucket or domain", it [appears to support](https://github.com/18F/dns/blob/a41fa2269f12a2498ace850604e65151b7b7eb48/terraform/18f.gov.tf#L656) redirection to a URL, though it adds a trailing slash.

In short: yes, though we may need to move to something else later.

**Comment by amirbey** [comment]
https://gsa-tts.slack.com/archives/GUW77KSGY/p1583350471000400
